packages/cli/src/databases/repositories/folder.repository.ts

```diff
@@ -293,4 +300,38 @@ export class FolderRepository extends Repository<FolderWithWorkflowAndSubFolderC
 			},
 		);
 	}
+
+	private applyExcludeFolderFilter(
+		query: SelectQueryBuilder<FolderWithWorkflowAndSubFolderCount>,
+		excludeFolderIdAndDescendants?: string,
```

Comment:
this is typed to accept `string | undefined`, but it should only accept `string`